.github/workflows/deployment.yml

```diff
@@ -1,5 +1,5 @@
 name: Deployment
-run-name: ${{ inputs.tag_name }} / go ${{ inputs.go_version }} / ${{ inputs.environment }}
+run-name: ${{ inputs.tag_name }} / go / ${{ inputs.environment }}
```

Comment:
Maybe just remove the `go` from here, but I'll wait till all the tests pass.

Reply:
Yeah, it isn't worth keeping that in the run name in reading it from `go.mod` 👍